moveit_core/robot_model/src/fixed_joint_model.cpp

```diff
@@ -36,63 +36,67 @@
 
 #include <moveit/robot_model/fixed_joint_model.h>
 
-moveit::core::FixedJointModel::FixedJointModel(const std::string& name) : JointModel(name)
+namespace moveit
+{
+namespace core
+{
+FixedJointModel::FixedJointModel(const std::string& name) : JointModel(name)
```

Comment:
add space after curly bracket and before function

Reply:
...unless clang-format does not allow this. do this for all the files in this PR please.

Reply:
Are you saying to do the change as following?
```
namespace moveit
{
namespace core
{

FixedJointModel::FixedJointModel(const std::string& name) : JointModel(name)
```

Reply:
Well, clang-format is not allowing to do the change i mentioned earlier.